### PR TITLE
feat: add node.js evaluator plugin

### PR DESCRIPTION
**Pull request by devhammed**
This pull request adds the ability to evaluate JavaScript (Node.js) expressions.

### CODE REVIEW

**Comment by ManuelSchneid3r** [comment]
time for reviews 

@BarbUk sorry i just wanted to try if addisigning outside collaborators as reviewers.

**Comment by BarbUk** [comment]
No problem

File: node_eval/__init__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import os
+import subprocess
+from albertv0 import *
+
+__iid__ = 'PythonInterface/v0.1'
+__prettyname__ = 'Node Eval'
+__version__ = '1.0'
+__trigger__ = 'node '
+__author__ = 'Hammed Oyedele'
+__dependencies__ = ['node']
```
Comment (ManuelSchneid3r) [inline node_eval/__init__.py:14]: please check the existance of the binary like so https://github.com/albertlauncher/python/blob/dev/units/__init_.py#L25-L26

File: node_eval/__init__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+iconPath = os.path.dirname(__file__) + '/nodejs.svg'
+
+
+def run(exp):
+    exp = exp.replace('"', '\\"')
+    return subprocess.getoutput(f'node --print "{exp}"')
```
Comment (ManuelSchneid3r) [inline node_eval/__init__.py:22]: please remove fstrings to get is working <3.6

File: node_eval/__init__.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+def handleQuery(query):
+    if query.isTriggered:
+        item = Item(id=__prettyname__, icon=iconPath, completion=query.rawString)
+        stripped = query.string.strip()
+
+        if stripped == '':
+            item.text = 'Enter a JavaScript expression...'
+        else:
+            item.text = run(stripped)
+            item.subtext = run(f'Object.prototype.toString.call({stripped}).slice(8, -1).toLowerCase()')
```
Comment (ManuelSchneid3r) [inline node_eval/__init__.py:34]: here as well

**Comment by ManuelSchneid3r** [comment]
👍

**Pull request merged**